### ISSUE TITLE
Fix out of the box MSSQL support

### ISSUE THREAD

**Issue by markmetcalfe** [totara/totara-docker-dev#128]
Before you can install totara on MSSQL, you must run 
```
EXEC sp_configure 'show advanced options', 1;
EXEC sp_configure 'clr strict security', 0;
RECONFIGURE;
GO
```
on the command line. We should make it so it automatically does it in the build.
In ```mssql/custom/setup.sql``` and ```mssql/custom/setup.sh``` it theoretically should do this, but it doesn't actually work in my experience.

**Comment by derschatta** [comment]
it already does it, see `mssql/custom/setup.sql`

**Comment by derschatta** [comment]
> it already does it, see `mssql/custom/setup.sql`

it should work

**Comment by derschatta** [comment]
you cannot do it in the build as the database needs to start up first.

**Comment by markmetcalfe** [comment]
> > it already does it, see `mssql/custom/setup.sql`
> 
> it should work

I found when I used a fresh container & volume it didn't work - I ran ```tstop mssql && tdocker rm mssql && docker volume rm docker-dev_mssql-data && tbuild mssql && tup mssql``` and then installed phpunit - and I got the clr strict security error.

```
Message: [Microsoft][ODBC Driver 17 for SQL Server][SQL Server]CREATE or ALTER ASSEMBLY for assembly 'GroupConcat' with the SAFE or EXTERNAL_ACCESS option failed because the 'clr strict security' option of sp_configure is set to 1. Microsoft recommends that you sign the assembly with a certificate or asymmetric key that has a corresponding login with UNSAFE ASSEMBLY permission. Alternatively, you can trust the assembly using sp_add_trusted_assembly.
```

**Comment by derschatta** [comment]
> > > it already does it, see `mssql/custom/setup.sql`
> > 
> > 
> > it should work
> 
> I found when I used a fresh container & volume it didn't work - I ran `tstop mssql && tdocker rm mssql && docker volume rm docker-dev_mssql-data && tbuild mssql && tup mssql` and then installed phpunit - and I got the clr strict security error.
> 
> ```
> Message: [Microsoft][ODBC Driver 17 for SQL Server][SQL Server]CREATE or ALTER ASSEMBLY for assembly 'GroupConcat' with the SAFE or EXTERNAL_ACCESS option failed because the 'clr strict security' option of sp_configure is set to 1. Microsoft recommends that you sign the assembly with a certificate or asymmetric key that has a corresponding login with UNSAFE ASSEMBLY permission. Alternatively, you can trust the assembly using sp_add_trusted_assembly.
> ```

did you get any output related to the setup.sql in your logs?

**Comment by derschatta** [comment]
```
totara_mssql                  | Running initial SQL statements
totara_mssql                  | 2020-09-30 00:32:16.06 spid51      Configuration option 'show advanced options' changed from 0 to 1. Run the RECONFIGURE statement to install.
Configuration option 'show advanced options' changed from 0 to 1. Run the RECONFIGURE statement to install.
totara_mssql                  | Msg 15123, Level 16, State 1, Server e4d9ac4d1398, Procedure sp_configure, Line 62
totara_mssql                  | The configuration option 'clr strict security' does not exist, or it may be an advanced option.
totara_mssql                  | 2020-09-30 00:32:16.10 spid9s      Polybase feature disabled.
```

that could explain it

**Comment by derschatta** [comment]
fixed with #129
